CONTRIBUTING.md

```diff
@@ -54,3 +54,28 @@ $ git tag -a -s -m vX.Y.Z vX.Y.Z <release-commit-ref>
 
 For this step, it is necessary to [configure Git to sign tags with SSH
 keys](https://docs.gitlab.com/ee/user/project/repository/signed_commits/ssh.html).
+
+## Pull Request Guidelines
+
+Ensure that each proposed commit is an operation that applies a set of distinct
+changes as a single operation (in other words, an atomic commit). The title
+should follow the pattern `<scope>: <description>.` The message should
+properly document why the changes should be done in the proposed way, present
+tradeoffs that were considered, alternative implementations considered and any
+other relevant information, such as co-authors, external references and related
+commits.
+
+Any information relevant for understanding the proposed changes should be
+contained in the respective commit messages, and only highlighted in the PR
+description for convenience. The pull request must include a brief description
+of the overall goal of the proposed changes, possibly including context
+information, motivations and related PRs or issues when relevant.
+
+Feel free to add other comments regarding the status of the proposed changes,
+e.g. what changes are still missing to promote the PR to non-draft or what
+needs further validation.
+
+Add a changelog entry (usually the commit title and the pull request URL) in
+[CHANGES.md](CHANGES.md) if the proposed changes are relevant for end-users.
+Use sub-items to provide concise extra information if required, following a
+similar format of existing entries.
```

Comment:
to provide relevant additional information in a concise manner, following the format of existing entries